studio/pages/account/me.tsx

```diff
@@ -78,13 +78,8 @@ const ProfileCard = observer(() => {
 })
 
 function ThemeSettings() {
-  const [theme, setTheme] = useState('')
   const { ui } = useStore()
-
-  useEffect(() => {
-    const localStorageTheme = window.localStorage.getItem('theme')
-    if (localStorageTheme) setTheme(localStorageTheme)
-  }, [])
+  const [value, setValue] = useState(ui.themeOption)
```

Comment:
likewise for here, no need to keep its own state, just listen directly to ui.themeOption